charts/barman/templates/deployment.yaml

```diff
@@ -81,6 +81,14 @@ spec:
           {{- end  }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+{{- if .Values.deployment.initContainers }}
+      initContainers:
+        {{- if eq (typeOf .Values.deployment.initContainers) "string" }}
```

Comment:
Can you quickly explain why we would be needing using a template for setting the initContainers? Is it not enough just rendering the array toYaml?

I see this construct also in the official velero Chart.

Reply:
I just copied pasted from Velero chart
Don't know the reason for that
Should I use only array ?

Reply:
Hmm I guess I know the reason
if you're templating over templating(e.g. helmfile gotmpl files), and you already has a string based value in other charts

Reply:
Makes sense